### PR TITLE
SWC-6147: change Dataset icon to match design

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Quick test in the demo app (updating one of the components to use DATASET instead of EXPERIMENTAL_TOOLS)
Screen Shot 2022-06-03 at 11 18 35 AM

File: src/lib/containers/row_renderers/utils/Icon.tsx [hunk 1]
```diff
@@ -42,6 +42,7 @@ import {
 } from '../../../utils/SynapseConstants'
 
 import { ReactComponent as Data2Svg } from '../../../assets/icons/Data2.svg'
+import { ReactComponent as DatasetSvg } from '../../../assets/icons/Dataset.svg'
```
Comment (nickgros) [inline src/lib/containers/row_renderers/utils/Icon.tsx:45]: I think this exists in MUI -- see '@material-ui/icons/GridOnTwoTone'. Can we use those in this Icon component?

File: src/lib/containers/row_renderers/utils/Icon.tsx [hunk 2]
```diff
@@ -138,26 +142,24 @@ const Icon: React.FunctionComponent<IconProps> = ({
...
 }) => {
+  const IconSvg = iconSvgMapping[value] || iconSvgMapping[type]
+  if (IconSvg) return IconSvg
+
   const iconSet = { ...defaultIcons, ...iconOptions }
   // see if the value has a corresponding icon, e.g. 'Active' in a studies table
   // or if the type of card has a corresponding icon, e.g. 'Publication'
-  const Icon = iconSet[value] || iconSet[type]
-  // TODO: get rid of dataset icon class, none of the icons should be special cased
-  const datasetCustomStyle =
```
Comment (jay-hodgson) [inline src/lib/containers/row_renderers/utils/Icon.tsx:146]: Removed custom styling for Dataset

File: src/lib/containers/row_renderers/utils/Icon.tsx [hunk 3]
```diff
@@ -138,26 +142,24 @@ const Icon: React.FunctionComponent<IconProps> = ({
   iconOptions,
   cssClass,
 }) => {
+  const IconSvg = iconSvgMapping[value] || iconSvgMapping[type]
```
Comment (jay-hodgson) [inline src/lib/containers/row_renderers/utils/Icon.tsx:145]: If this type/value is supported by our IconSvg mapping above, then return it.

**Comment by jay-hodgson** [comment]
Uses MUI icon instead.
Screen Shot 2022-06-03 at 3 44 03 PM

**Pull request merged**